base/utils.sh

```diff
@@ -36,6 +37,12 @@ install_local_deps() {
     done
 }
 
+install_test_deps() {
+    if [ "$INSTALL_TESTS" = "True" ]; then
+        python3 /opt/oci_env/base/container_scripts/install_test_requirements.py
```

Comment:
Instead of using a python script I would just copy-paste the for loop above and change the pip command to install from `test_requirements.txt`.

Reply:
I did this, but it did not help:
```
install_test_deps() {
    if [ "$INSTALL_TESTS" = "True" ]; then
        local src_path_list
        IFS=':' read -ra src_path_list <<< "$DEV_SOURCE_PATH"

        for item in "${src_path_list[@]}"; do
            src_path="/src/${item}"
            if [[ -d "$src_path" ]]; then
                log_message "Installing requirements for ${item}."
                pip3 install -r "${src_path}"/lint_requirements.txt || true
                pip3 install -r "${src_path}"/unittest_requirements.txt || true
                pip3 install -r "${src_path}"/functest_requirements.txt || true
                pip3 install -r "${src_path}"/perftest_requirements.txt || true
            fi
        done
        # python3 /opt/oci_env/base/container_scripts/install_test_requirements.py
    fi
}
```
I am getting the same error. And, I know the error is related to my changes because when I removed the `install_test_deps` call from `init_container`, it worked.